feat(aps): rework troubleshooting (#4338)
scaleway/docs-content: pages/apple-silicon/troubleshooting/mac-mini-installation-issues.mdx

---
meta:

>   title: Troubleshooting Mac mini Installation Issues

Reviewer: ```suggestion
  title: Troubleshooting Mac mini installation issues
```

>   description: Learn how to resolve common installation problems with Scaleway Apple Silicon Mac mini.

Reviewer: ```suggestion
  description: Learn how to resolve common installation problems with Scaleway Apple silicon Mac mini.
```

content:
  h1: Troubleshooting Mac mini Installation Issues

>   paragraph: Learn how to resolve common installation problems with Scaleway Apple Silicon Mac mini.

Reviewer: ```suggestion
  paragraph: Learn how to resolve common installation problems with Scaleway Apple silicon Mac mini.
```

tags: installation, setup, mac-mini
dates:
  validation: 2025-01-31
  posted: 2025-01-31
categories:
  - bare-metal
---

<Macro id="requirements" />
- A Scaleway account logged into the [console](https://console.scaleway.com)
- [Owner](/iam/concepts/#owner) status or [IAM permissions](/iam/concepts/#permission) allowing you to perform actions in the intended Organization
- An [SSH key](/organizations-and-projects/how-to/create-ssh-key/)
- A [Mac mini](/apple-silicon/how-to/create-mac-mini/)

## Problem
The installation of your Mac mini failed.

### Symptoms

> The installation of your hosted Mac mini could not complete successfully.

Reviewer: ```suggestion
The installation of your hosted Mac mini could not be completed successfully.
```

### Possible causes
- Incompatible installation image.
- Network interruptions during installation.
- Hardware compatibility issues.

### Solution
1. Verify installation image:

>    - Ensure that the installation image is not is compatible with your Mac mini version. If unsure try a different version and relaunch the installation process.

Reviewer: ```suggestion
   - Ensure that the installation image is compatible with your Mac mini version. If unsure, try a different version and relaunch the installation process.
```
